scripts/store/cart.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * scripts/store/cart.js
+ * Does update logic for cart quantities and some basic address validation
+ */
+
+if (typeof country === 'undefined' || country.length === 0) {
+    console.error('Unable to find country data')
+
+    var country = null
+    $.getJSON('data/country.json', function (data) {
+        console.log('Was able to fetch country data manually')
+
+        country = data
+    })
+}
+
+/**
+ * updateTotal
+ * Adds prices for everything in cart and puts sub total in footer
+ */
+var updateTotal = function () {
+    var $products = $('.list--product .list__item')
+
+    if ($products.length <= 0) location.reload()
+
+    var total = 0
+
+    $products.each(function (i, p) {
+        var n = $(p).attr('id').replace('product-', '')
+        var $i = $('.list__item#product-' + n)
+
+        var price = $('input[name$="price"]', $i).val()
+        var quantity = $('input[name$="quantity"]', $i).val()
+
+        var t = (price * quantity)
+        $('.subtotal b', $i).text('$' + parseFloat(t).toFixed(2))
+
+        total += t
+    })
+
+    $('.list--product .list__footer h4').text('Sub-Total: $' + parseFloat(total).toFixed(2))
+}
+
+/**
+ * POSTs to inventory endpoint to update cart quantities without page refresh
+ */
+$('.list--product .list__item input[name$="quantity"]').on('change', function (e) {
```

Comment:
We should also do this after a timeout of no entry, probably around 500ms. We do this on the System76 cart because it's not obvious that unfocusing the entry will update live, so people end up hitting enter.